components/ProfileImage.vue

```diff
@@ -77,4 +77,14 @@ export default {
     height: 50px;
   }
 }
+
+.profile--xl {
+  width: 100px;
+  height: 100px;
+
+  @include media-breakpoint-up(md) {
```

Comment:
Feels slightly odd to have a media query setting the same value as the default

Reply:
The image was 100px in all breakpoints before but I've now changed it to be 50px on smaller screens and 100px on larger screens.  This whole section needs some work though as there are existing layout issues that I haven't touched for now.